src/dotnet/UI.Blazor/Components/Navbar/LeftPanel/WideLeftPanel.razor

```diff
@@ -34,9 +35,14 @@
         }
     }
 
-    private void Click() {
+    private void OnClick() {
         if (NavbarUI.AddButtonAction.TryGetValue(NavbarUI.ActiveGroupId, out var addAction)) {
             addAction();
         }
     }
+
+    private void OnSearch(string? criteria) {
+        SearchUI.Criteria.Value = criteria ?? "";
+    }
+
```

Comment:
We normally don't add extra empty lines in such cases.

Reply:
fixed